extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestBase.java

```diff
@@ -807,12 +809,34 @@ private MSQErrorReport getErrorReportOrThrow(String controllerTaskId)
 
   private void assertMSQSpec(MSQSpec expectedMSQSpec, MSQSpec querySpecForTask)
   {
-    Assert.assertEquals(expectedMSQSpec.getQuery(), querySpecForTask.getQuery());
+    assertMSQSpecQuery(expectedMSQSpec.getQuery(), querySpecForTask.getQuery());
     Assert.assertEquals(expectedMSQSpec.getAssignmentStrategy(), querySpecForTask.getAssignmentStrategy());
     Assert.assertEquals(expectedMSQSpec.getColumnMappings(), querySpecForTask.getColumnMappings());
     Assert.assertEquals(expectedMSQSpec.getDestination(), querySpecForTask.getDestination());
   }
 
+  private void assertMSQSpecQuery(Query msqSpecQuery, Query taskSpecQuery)
```

Comment:
```suggestion
  private void assertMSQSpecQuery(Query expectedQuery, Query taskSpecQuery)
```

Reply:
Have reverted this code block, so this is a non-issue now.